src/pivotal-ui/components/forms/link-updater.js

```diff
@@ -14,7 +14,7 @@ var LinkUpdater = function(el){
 LinkUpdater.prototype.updateValue = function(inputValue){
   var pathVal = inputValue;
   if(inputValue === ""){
-    pathVal = "username";
+    pathVal = this.input.attr("placeholder") || "username";
```

Comment:
Interesting. Means that the association isn't dynamic, though -- it means updating the placeholder won't be reflected.